samples/loganalytics/manage_tables_list_by_workspace.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.loganalytics import LogAnalyticsManagementClient
+
+
+def main():
+    subscription_id = os.environ["AZURE_SUBSCRIPTION_ID"]
+    credentials = DefaultAzureCredential()
+    GROUP_NAME = "test"
+    workspace_name = "testWorkspace"
+    resource_client = ResourceManagementClient(
+        credential=DefaultAzureCredential(),
+        subscription_id=subscription_id
+    )
+    log_analytics_client = LogAnalyticsManagementClient(credentials, subscription_id, )
+    #
+    loganalytics_client = LogAnalyticsManagementClient(credentials, subscription_id, )
+    resource_client.resource_groups.create_or_update(
+        GROUP_NAME,
+        {"location": "eastus"}
+
+    )
+
+    log_analytics_client.workspaces.begin_create_or_update(
+        resource_group_name='zb_test',
+        workspace_name='oiautorest6685',
+        parameters={
+            "location": "australiasoutheast",
+            "properties": {"retentionInDays": 30, "sku": {"name": "PerGB2018"}},
+            "tags": {"tag1": "val1"},
+        },
+    ).result()
+
+    response = log_analytics_client.tables.list_by_workspace(
+        resource_group_name='zb_test',
+        workspace_name='oiautorest6685'
+    )
+    for item in response:
+        print(item)
```

Comment:
If you want to see all the items in model, please see https://github.com/Azure-Samples/azure-samples-python-management/commit/8405a54d62cddd9b5090092a9998550e4f305763#diff-518810fc5dbce2ba097d1efe6b69c53f91767c6f94d8ebf41918acfbbac7b3ad:
![image](https://user-images.githubusercontent.com/70930885/211280467-a3563952-06b5-4ad6-a29b-27abd50bb980.png)
It is more friendly for users.